src/services/nnid/routes/people.ts

```diff
@@ -580,7 +580,8 @@ router.put('/@me', async (request: express.Request, response: express.Response):
 	const region: number = person.region ? person.region : pnid.region;
 	const countryCode: string = person.country ? person.country : pnid.country;
 	const language: string = person.language ? person.language : pnid.language;
-	const timezoneName: string = person.tz_name ? person.tz_name : pnid.timezone.name;
+	// The 3DS sends an empty object when the timezone isn't updated, so this fixes validation from failing.
+	const timezoneName: string = (person.tz_name && !!Object.keys(person.tz_name).length) ? person.tz_name : pnid.timezone.name;

```

Comment:
I can't seem to replicate this? But that could because my account launguage is spanish, and I remember that for some people that the NNID settings worked better on spanish?

Reply:
I encountered this while testing on my 3DS locally. It might have been a one time bug, but it was consistently sending a empty tag when the timezone wasn't changed. I didn't check to see if this continued after restarting the app.